server/acl_casbin_policy_dba.csv

```diff
@@ -87,3 +87,8 @@ p, DBA, /savedquery/{id}, PATCH_SELF
 p, DBA, /savedquery/{id}, DELETE_SELF
 p, DBA, /subscription, GET
 p, DBA, /subscription, PATCH
+p, DBA, /sheet, POST
+p, DBA, /sheet, GET
+p, DBA, /sheet/{id}, GET
+p, DBA, /sheet/{id}, PATCH_SELF
```

Comment:
This will prevent non-owner updating the sheet. It's an acceptable behavior as the starting point, but is a bit different from our off-line discussion. Just FYI.